ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh

```diff
@@ -41,6 +41,21 @@ function prepare_next_steps() {
       "${dir}/auth/kubeconfig" \
       "${dir}/auth/kubeadmin-password" \
       "${dir}/metadata.json"
+  
+  # For private cluster, the bootstrap address is private, installer cann't gather log-bundle directly even if proxy is set
+  # the workaround is gather log-bundle from bastion host
+  # copying install folder to bastion host for gathering logs
+  set -x
+  publish=$(grep "publish:" ${SHARED_DIR}/install-config.yaml | awk '{print $2}')
+  if [[ "${publish}" == "Internal" ]] && [[ ! $(grep "Bootstrap status: complete" "${dir}/.openshift_install.log") ]]; then
+    echo "Copying install dir to bastion host."
+    bastion_ssh_user=$(head -n 1 "${SHARED_DIR}/bastion_ssh_user")
+    bastion_public_address=$(head -n 1 "${SHARED_DIR}/bastion_public_address")
+    cmd="scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o VerifyHostKeyDNS=yes  -i \"${CLUSTER_PROFILE_DIR}/ssh-privatekey\" -r ${dir} ${bastion_ssh_user}@${bastion_public_address}:/tmp/"
```

Comment:
how about append a hardcode name for installer working dir - `/tmp/installer`

Reply:
The `dir` was defined in https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh#L102

Reply:
Yeah, that is the source dir name, I meant we can hardcode the target folder name, so that keep consistent with the expected path in `gather-installer-boostrap-logs-in-bastion-commands.sh`, but not impacted by any change of `dir` in the future (though it is almost not changed often).